### DISCUSSION TITLE
15min consumption points

### DISCUSSION THREAD

**Discussion by gsaurer** [DarwinsBuddy/WienerNetzeSmartmeter#42] [General]
You can configure that the smart meter is sending the consumption in the 15min interval. Is there a plan to take this into account in the extension?

**Comment by DarwinsBuddy** [comment]
@reox is currently working on a way but it's really hard to connect Winer Netze API
to HomeAssistant in this way. (WN API has some limitations which are somewhat seemingly incompatible)

if you're interested in the details see https://github.com/DarwinsBuddy/WienerNetzeSmartmeter/issues/8

:)

**Comment by hirschy81** [comment]
+1 for this feature!

**Comment by DarwinsBuddy** [comment]
rolled out a while ago ;)
